server/src/main/java/com/networknt/server/Server.java

```diff
@@ -396,7 +396,7 @@ protected static KeyStore loadTrustStore() {
                 throw new RuntimeException("Unable to initialise TrustManager[]", e);
             }
         } else {
-            logger.warn("Unable to find server truststore while Mutual TLS is enabled. Falling back to trust all certs.");
+            // Mutual Tls is disabled, trust all the certs
             trustManagers = TRUST_ALL_CERTS;
```

Comment:
When Mutual TLS is enabled, I think we should still display that warn message.

Reply:
Hi!

This function is invoked like:
```
if (getServerConfig().isEnableTwoWayTls()) {
     trustManagers = buildTrustManagers(loadTrustStore());
} else {
      trustManagers = buildTrustManagers(null);
}
```
Therefore, if the Mutual TLS is enabled, the `logger` line would not be executed. 

For `loadTrustStore()` method, it would not return `null` due to the changes in Config.class. When server.truststore cannot be found, an ConfigException would throw, which contains the reason: "Unable to load 'server.truststore', please provide the keystore/truststore matching the configuration in client.yml/server.yml to enable TLS connection."

Beside the exception, the error also be logged as : 
```
logger.error("Unable to load truststore " + name, e);
```

The keystore has the same behaviour.

Do you means that when enable TMLS without truststore, just warning and trust all the cert instead of throw an exception?  Would that lead to some security problems?

Reply:
> Do you means that when enable TMLS without truststore, just warning and trust all the cert instead of throw an exception? Would that lead to some security problems?

No, that is how it works now but I don't think it's a good practice.

Currently there still could be issues if users choose different extensions for their keystrokes, like .jks for example. Could we make this independent of what the filename for the keystone actually is?

Reply:
Thanks for the reply!
This has been fixed. I put the keystore and truststore loading into TlsUtil.class for centralized management. In this way, the client and server will have consistent performance when loading the keystore and truststore. When loading a keystore or truststore, regardless of any extensions, an exception is thrown as long as the input stream cannot be obtained by its name.